core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.spec.ts

```diff
@@ -540,13 +540,6 @@ describe("SavedWorkflowSectionComponent", () => {
     expect(component.downloadListWorkflow.get(index)).toEqual("workflow-2.json");
   });
 
-  it("editing a workflow name triggers a POST request on the backend", () => {
```

Comment:
Why would you delete this test?

Reply:
Sorry for late reply. I realized this test is actually very trivial and not as useful as the one for the workflow description. And it was actually added by me in a previous PR which I only did out of testing purpose and forgot to delete.